templates/tower-project.yaml

```diff
@@ -186,7 +186,7 @@ Resources:
   EncryptionKeyStack:
     Type: AWS::CloudFormation::Stack
     Properties:
-      TemplateURL: !Sub ${TemplateRootUrl}/aws-infra/v0.3.1/KMS/kms-key.yaml
+      TemplateURL: !Sub ${TemplateRootUrl}/KMS/kms-key.yaml
```

Comment:
@ahayden: I see the need to use the latest template for security reasons. However, I am afraid of deploying an unrelated change and things start breaking because the KMS key template works differently after an update. How should we balance these two goals?

Reply:
This is really just a style choice... My comment was meant so that you would pass in the library version with the `TemplateRootUrl` like 'https://remoterepo.tld/aws-infra/v0.3.1/'... and so would only have to change the parameter when if you wanted to inc library versions